cdap-ui/app/cdap/components/Experiments/CreateView/SplitDataStep/SplitInfoTable/index.js

```diff
@@ -159,9 +181,22 @@ export default class SplitInfoTable extends Component {
             return (
               <tr
                 key={field.name}
+                className={classnames({
+                  'active': field.name === this.state.activeColumn
+                })}
                 onClick={this.props.onActiveColumnChange.bind(null, field.name)}
               >
-                <td>{field.name}</td>
+                 <td>
+                  {
+                    this.state.outcome === field.name ? (
+                      <span className="outcome-field">
+                        <IconSVG name="icon-star" />
+                        <span> {field.name} </span>
+                      </span>
+                    )
+                    : field.name
+                  }
+                </td>
```

Comment:
Do you think we can merge `renderNumericalTableBody()` and `renderCategoricalTableBody()`? Seems like they're very similar right now, besides some `<td>` fields. Maybe we can conditionally render those fields depending on whether the data is categorical or numerical? If you think they're gonna differ significantly in the future then don't worry about it.

Reply:
I would prefer to keep it separate. Its easier to debug and understand. Its the trade-of between having a more distinct path for different things vs abstraction to some level.